### ISSUE TITLE
What was the dataset used for testing in the paper?

### ISSUE THREAD

**Issue by edunov** [tensorflow/tensor2tensor#226]
Hi,
In the paper, you mention that the dataset used for testing is newstest2014: https://arxiv.org/abs/1706.03762

While in the code, I can only see references to newstest2013 for both En2Fr and En2De:

https://github.com/tensorflow/tensor2tensor/blob/45a787e46b32bdb18b70f835cba0b3270267e19b/tensor2tensor/data_generators/wmt.py#L208
https://github.com/tensorflow/tensor2tensor/blob/45a787e46b32bdb18b70f835cba0b3270267e19b/tensor2tensor/data_generators/wmt.py#L238

Is the code out of date?

**Comment by lukaszkaiser** [comment]
Hi. No, the code is fine: we only include the development (evaluation) set in our prepared data, never the training set. We only run once on the training set, after training the best model. And we only run decoding, so we only need the test set in the raw form (detokenized).

**Comment by lukaszkaiser** [comment]
I mean: never the test set, sorry.
